server/sql/dev/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+
+services:
+  mongodb:
+    image: mongo
+    ports:
+      - "27017:27017"
+
+  mongo-seed-user:
+    image: mongo
+    links:
+      - mongodb
+    volumes:
+      - ./:/seed-data
+    command:
+      'bash -c "mongoimport --host mongodb --db socobo_dev --collection socobousers --type json --file /seed-data/create-socoboUser.json --jsonArray"'
```

Comment:
i think the socobouser collection 'socobousers' should fail, because have named it 'socobouser' everywhere.

Reply:
Oh the file in /dev is the wrong one. You need to use the one directly in sql. There I use socobouser as collection. I remove the dev folder I think we don't need it anymore since all data are hold in /data